### ISSUE TITLE
Pink Elephant

### ISSUE THREAD

**Issue by seanneumann** [opensearch-project/oui#521]
**Is your feature request related to a problem? Please describe.**

I'd like to build an OUI diagnostics theme called Pink Elephant. Let's change all the OUI styles to be pink to highlight where in OpenSearch Dashboards and plugins that aren't using oui. Basically, anything that is not pink is not using OUI.

It should be deployable to staging domain: pink.playground.opensearch.org

Related Playground Issue: https://github.com/opensearch-project/dashboards-anywhere/issues/158

**Comment by KrooshalUX** [comment]
@BSFishy  Lets try these colors for a first round - may need to make some edits once I see in staging:

// Global Colors
`$ouiColorGhost: #FFFCFD;`
` $ouiColorInk: #3f1497;`

  // Core 
`$ouiColorPrimary: #ff009d;`
` $ouiColorSecondary: #6448d6;`
` $ouiColorAccent: #bada55;`

  // Status
` $ouiColorSuccess: $ouiColorSecondary; `
`$ouiColorWarning: #ffb2e2;`
 `$ouiColorDanger: #Ff3db4;`

  // Grays
` $ouiColorEmptyShade: #FFFCFD;`
` $ouiColorLightestShade: #F5C1E2;`
` $ouiColorLightShade: #DF97C4;`
` $ouiColorMediumShade: #D982B7;`
` $ouiColorDarkShade: #D575B0;`
` $ouiColorDarkestShade: #D167A8;`
` $ouiColorFullShade: #CC589F;`

  // Backgrounds
` $ouiPageBackgroundColor: shade($ouiColorLightestShade, 45%); `
`$ouiColorHighlight: #FFDDDD;`

  // Variations from core
` $ouiTextColor: #3D1495; `

// Shadows 
`$ouiShadowColor: #000; `
`$ouiShadowColorLarge: #000;`

**Comment by KrooshalUX** [comment]
@BSFishy round 2:

  // Core 
$ouiColorPrimary: #CFB3FF;
$ouiColorSecondary: #FFD481;
$ouiColorAccent: #9DCD65;

  // Status
$ouiColorSuccess: $ouiColorSecondary; 
$ouiColorWarning: #FFCBC3;
$ouiColorDanger: #F57600;

  // Grays
$ouiColorEmptyShade: #FDDFEC;
$ouiColorLightestShade: #FBD2E2;
$ouiColorLightShade: #F8B8D0;
$ouiColorMediumShade: #F194B8;
$ouiColorDarkShade: #E6308A;
$ouiColorDarkestShade: #CD2576;
$ouiColorFullShade: #B51963;

**Comment by joshuarrrr** [comment]
@KrooshalUX I know the original scope of this is for theming, which is primarily about colors. But do we wand to also take a similar approach to identify other gaps, such as font stacks?

**Comment by KrooshalUX** [comment]
@joshuarrrr yes - I am working on identifying some additional changes we can make to the other style driven elements. One of our biggest questions however, is if theming is overriding the overrides, as much more is "pink" than we expected. There is a meeting next week to come up with additional investigation and next steps. Stay tuned!

**Comment by anirudha** [comment]
Closing — this discussion/proposal has been inactive for over a year with no traction. If there's renewed interest, a fresh issue with updated context would be more productive.
